components/rating/rating.css

```diff
@@ -0,0 +1,68 @@
+.star-wrapper input{
+    display: none;
+  }
+  .star-example{
+    border: 1px solid var(--primary-dark-orange);
+    height: 15rem;
+  }
+  .star-wrapper  label{
+    float: right;
+    font-size: 5rem;
+    color: var(--primary-faint-grey);
+    margin: 0 0.2rem;
+    text-shadow: 1px 1px var(--primary-faint-grey);
+  }
+  .star-wrapper  label:before{
+    content: '★';
+    cursor: pointer;
+  }
+  .star-wrapper{
+    height: 7rem;
+    width: 25rem;
+    position: relative;
+  }
+  .star-wrapper input:checked ~ label{
+    color: var(--primary-gold);
+    text-shadow: 1px 1px var(--primary-dark-orange);
+  }
+  .star-wrapper:not(:checked) > label:hover,
+  .star-wrapper:not(:checked) > label:hover ~ label{
+    color: var(--primary-gold);;
+  }
+  .star-wrapper input:checked > label:hover,
+  .star-wrapper input:checked > label:hover ~ label{
+    color: var(--primary-gold);;
+    text-shadow: 1px 1px goldenrod;
+  }
```

Comment:
Does text-shadow works??

Reply:
yes it gives a little color to the stars